apps/visualize_network.py

```diff
@@ -21,6 +24,9 @@
     visualizer.update_losses(2., 7., type='validation')
     visualizer.update_losses(4., 8., type='validation')
     visualizer.update_losses(6., 9., type='validation')
+    for batch, in loader:
+        visualizer.plot_batch(batch, ['img1', 'img2', 'img3', 'img4'])
+        break
```

Comment:
break here looks suspicious

Reply:
That's just a test visualization script. I want to visualize 1 batch from dataloader to make sure visualization is correct